servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilder.java

```diff
@@ -96,7 +100,11 @@ final class DefaultMultiAddressUrlHttpClientBuilder
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultMultiAddressUrlHttpClientBuilder.class);
 
-    private static final String HTTPS_SCHEME = HTTPS.toString();
+    // Use HttpRequestMetaData to access "https" constant used by Uri3986 class to optimize "equals" check to be a
+    // trivial reference check.
```

Comment:
Clever. 😄

Reply:
The `selectClient` was about 10% of CPU on the flame graph. There will be a few more follow-ups in this area, I'm just trying to isolate independent changes in separate PRs